openslides/assignments/static/js/assignments/site.js

```diff
@@ -527,20 +528,27 @@ angular.module('OpenSlidesApp.assignments.site', [
         };
         // create new ballot
         $scope.createBallot = function () {
-            $scope.activeTab = 0;
             $http.post('/rest/assignments/assignment/' + assignment.id + '/create_poll/').then(
                 function (success) {
                     $scope.alert.show = false;
                     if (assignment.phase === 0) {
                         $scope.updatePhase(1);
                     }
+                    $timeout(function() {
```

Comment:
Yes, we need a timeout to select a recently created tab, see https://github.com/angular-ui/bootstrap/issues/5656. Otherwise the last tab is not selected. 
For delete tab the timeout is not required (I tested it).